src/main/java/server/response/ResponseBuilder.java

```diff
@@ -2,16 +2,25 @@
 
 import server.request.Request;
 
-import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
 
 import static server.Routes.ROUTES;
 
 public class ResponseBuilder {
     public static Response buildResponse(Request request) {
-        if (Arrays.asList(ROUTES).contains(request.path())) {
-            return new Response(StatusCodes.OK, ReasonPhrases.OK);
+        Map<String, String> route = ROUTES.get(request.path());
+
+        if (route != null) {
+            return new Response(StatusCodes.OK, ReasonPhrases.OK, buildHeaders(route));
         } else {
-            return new Response(StatusCodes.NOT_FOUND, ReasonPhrases.NOT_FOUND);
+            return new Response(StatusCodes.NOT_FOUND, ReasonPhrases.NOT_FOUND, new HashMap<String, String>());
```

Comment:
Just a tip: it's generally best practice to use something like `Collections.emptyMap` here, which will avoid creating a  new `HashMap` every time. 

https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptyMap--

Reply:
ooh nice, thanks!